lib/graphiti/resource/interface.rb

```diff
@@ -50,6 +50,11 @@ def validate!(params)
           end
         end
       end
+
+      included do
```

Comment:
Is there a reason for the `included` hook, as opposed to just `def`? Should add as an instance method either way?

Reply:
👍